Extensions/PathfindingBehavior/pathfindingruntimebehavior.ts

```diff
@@ -479,24 +477,25 @@ namespace gdjs {
       _rightBorder: integer = 0;
       _topBorder: integer = 0;
       _bottomBorder: integer = 0;
-      _distanceFunction: any;
-      _allNodes: any = [];
+      _distanceFunction: Function;
```

Comment:
Let's type this entirely for more safety. Something like `(pt1: FloatPoint, pt2: FloatPoint) => float` I guess.